examples/src/utils/renderer-debug-widget.ts

```diff
@@ -1,396 +0,0 @@
-/*!
- * Copyright 2021 Cognite AS
- */
-
```

Comment:
Could be ported to rendering visual tests because otherwise will probably be forgotten forever and reimplemented in the future. Also can be helpful in testing primitives vs meshes performance.